src/main/scala/streamer/TestParameter.scala

```diff
@@ -16,3 +16,27 @@ object UnrollingAddrGenUintTestParameters {
   def addrWidth = 32
 
 }
+
```

Comment:
@xiaoling-yi 

It would be cool if you have a documentation comment at the top of the file indicating what each parameter controls or tells us :) for documentation purposes.

I mean I would understand cause I know some but what about those who don't know what each parameter does?

Something like...

```suggestion
//-----------------------------
// Parameter definitions
//-----------------------------
// dataReaderTcdmPorts - Indicates the number of connections to TCDM ports
// tcdmDataWidth - indicates data width for each TCDm port
// unrollingFactor - spatial unrolling (your parfor)
// ... etc.
//-----------------------------
```